tvbwidgets/ui/ts_widget.py

```diff
@@ -289,7 +313,6 @@ def _create_checkboxes(self, array_wrapper):
         unselect_all_btn.on_click(self._unselect_all)
 
         # select dimensions region
-        # TODO position for Sv and mode should come from Wrapper
```

Comment:
As discussed, the user will be instructed that the input should always have the State variable on position 1 and the Mode on position 3, so there is no need to compute their positions in the Wrapper.